bittensor/utils/__init__.py

```diff
@@ -20,16 +20,18 @@
 
 import bittensor
 import hashlib
-import requests
 import torch
 import scalecodec
 
 from .wallet_utils import *  # noqa F401
+from .version import version_checking
 
 RAOPERTAO = 1e9
 U16_MAX = 65535
 U64_MAX = 18446744073709551615
 
+VERSION_CHECK_THRESHOLD = 86400
+
```

Comment:
Duplicate, already exists in `version.py`

Reply:
Oh, missed that one when extracting everything to a separate module